### ISSUE TITLE
Stage Manager with multiple monitors

### ISSUE THREAD

**Issue by rusaito** [rxhanson/Rectangle#928]
### Discussed in https://github.com/rxhanson/Rectangle/discussions/927

<div type='discussions-op-text'>

<sup>Originally posted by **rusaito** September 25, 2022</sup>
I have a three-monitor set-up, and the stage manager (ventura beta) recent app area (I set to 175px) is correctly on the left side for my (1st) middle, and (2nd) left monitor. My (3rd) right monitor has the stage manager recent app area buffer on the right side instead of the left. In other words the windows default to cover up the stage manager and leave the same 175px buffer on the opposite (right side) </div>

**Comment by lzilioli-prism** [comment]
I have this same issue with a 2 monitor setup. Also only a problem on my right-most display.

**Comment by therealFoxster** [comment]
> I have this same issue with a 2 monitor setup. Also only a problem on my right-most display.

I'm also experiencing the same issue.

**Comment by rxhanson** [comment]
I’ll plan to take a look at this one for the next release.

**Comment by lzilioli-prism** [comment]
Is this in the latest release or does a new version need to be published first? Looking forward to trying out the fix!

**Comment by rxhanson** [comment]
It hasn’t been released yet. You could try out the latest auto-generated build if you want to try it sooner. Auto generated builds are not signed so you would have to bypass gatekeeper for them.

**Comment by lzilioli-prism** [comment]
Tested it with the new version today. Works for me!

One thing that I would like to see as an enhancement (and I will file a separate suggestion for this) is the ability to NOT cover the apps in the stage when I use the hotkeys. I prefer to see the apps on my stage manager sidebar at all times. In ventura, my version of "maximizing" a window should reserve space for the apps. In other words, I did like the previous behavior where rectangle honored the gap where the staged apps live, it was just an issue that on the right-most monitor Rectangle was determining the gap to be on the wrong side of the display.
